streaming/base/stream.py

```diff
@@ -327,6 +327,11 @@ def _decompress_shard_part(self, zip_info: FileInfo, zip_filename: str, raw_file
 
         # Validate what was downloaded.
         if self.validate_hash:
+            if self.validate_hash not in zip_info.hashes:
+                raise ValueError(
+                    f'Hash algorithm `{self.validate_hash}` provided for data ' +
```

Comment:
First "provided" -> chosen (so we have a different verb than the second provided)?
"do not" -> "does not"

Reply:
Done.